### ISSUE TITLE
Regression on 0.12.0

### ISSUE THREAD

**Issue by sylvainpolletvillard** [cjpatoilo/banner-cli#2]
Hello,

Upgrading to v0.12.0 breaks banner-cli on my machine (Windows 10x64, Node 8.2.1, npm 5.6.0).

The problem is that the build command `banner-cli dist/*.js` modifies the banner-cli source files itselves.

If I check `node_modules\banner-cli\cli.js` file after running the build command, I see :
```javascript
/*!
 * Objectmodel v3.2.2
 * http://objectmodel.js.org
 *
 * Copyright (c) 2018 Sylvain Pollet-Villard
 * Licensed under the MIT license
 */

#!/usr/bin/env node
'use strict'

const cli = require('./index')
const pkg = require('./package.json')
const argv = require('rasper')()
let options = {}

if (argv.v || argv.version) {
	console.log(pkg.version)
	process.exit(0)
}

if (argv.h || argv.help) {
	console.log(`
  Usage:

    $ banner-cli <files> [<options>]
...
```

As you can see, the banner of my project has been added here, causing an error because `#!/usr/bin/env node` must be the first line of the file:

```
W:\github\ObjectModel\node_modules\banner-cli\cli.js:9
#!/usr/bin/env node
^

SyntaxError: Invalid or unexpected token
```

Rollbacking to 0.11.1 solves the problem.

If you need to test on my project, you can clone it from here : https://github.com/sylvainpolletvillard/ObjectModel

Thanks !

**Comment by dezudas** [comment]
I faced the same issue and rolled back to 0.11.1 which is working now.

**Comment by fabiommendes** [comment]
It works here if we change the last option to receive `argv._[1]` instead of `argv._[0]`.  Maybe the actual solution is more complicated if different versions of node pass the current script at different positions in the argv array. I guess that if the author published the script this way it must work at least in *some* node environments. I'm using node v11.14 and it shows the same issue you reported.

**Comment by cjpatoilo** [comment]
@fabiommendes was right. The bug has been fixed in the new version `v0.13.0`.
BanneCLI was also upgraded to support the new versions of the Node.js.

Guys, thank you a lot.
